docs/writing-rust-tas-using-optee-utee-build.md

```diff
@@ -0,0 +1,183 @@
+---
+permalink: /trustzone-sdk-docs/writing-rust-tas-using-optee-utee-build.md
+---
+
```

Comment:
When finalizing this documentation, please rewrap all text in this document at 80 characters, this helps to further review and see changes.
Reference: https://thoughtbot.com/blog/wrap-existing-text-at-80-characters-in-vim